### PR TITLE
Add nuget packages

### PR DESCRIPTION
**Pull request by v-wuzhai**
Added the following nuget packages to fix [#7041](https://github.com/dotnet/templating/pull/7041)

> NuGet.Common.6.7.0
> NuGet.Configuration.6.7.0
> NuGet.Credentials.6.7.0
> NuGet.Frameworks.6.7.0
> NuGet.Packaging.6.7.0
> NuGet.Protocol.6.7.0
> NuGet.Versioning.6.7.0

### CODE REVIEW

File: src/referencePackages/src/system.runtime/4.3.0/ref/net45/_._
Comment (mthalman) [inline src/referencePackages/src/system.runtime/4.3.0/ref/net45/_._:1]: This should be reverted.

File: eng/Build.props [hunk 1]
```diff
@@ -28,6 +28,14 @@
...
     <DependencyPackageProjects Include="$(RepoRoot)src\targetPacks\ILsrc\**\Microsoft.NETCore.App.Ref.6.0.2.csproj" />
     <DependencyPackageProjects Include="$(RepoRoot)src\targetPacks\ILsrc\**\Microsoft.AspNetCore.App.Ref.6.0.2.csproj" />
+
+    <DependencyPackageProjects Include="$(RepoRoot)src\referencePackages\src\**\NuGet.Common.6.7.0.csproj" />
+    <DependencyPackageProjects Include="$(RepoRoot)src\referencePackages\src\**\NuGet.Configuration.6.7.0.csproj" />
+    <DependencyPackageProjects Include="$(RepoRoot)src\referencePackages\src\**\NuGet.Credentials.6.7.0.csproj" />
+    <DependencyPackageProjects Include="$(RepoRoot)src\referencePackages\src\**\NuGet.Frameworks.6.7.0.csproj" />
+    <DependencyPackageProjects Include="$(RepoRoot)src\referencePackages\src\**\NuGet.Packaging.6.7.0.csproj" />
+    <DependencyPackageProjects Include="$(RepoRoot)src\referencePackages\src\**\NuGet.Protocol.6.7.0.csproj" />
+    <DependencyPackageProjects Include="$(RepoRoot)src\referencePackages\src\**\NuGet.Versioning.6.7.0.csproj" />
```
Comment (mthalman) [inline eng/Build.props:38]: This need to be listed in dependency order. For example, NuGet.Common has a dependency on NuGet.Frameworks. So NuGet.Frameworks should be listed before NuGet.Common. Here is a PR that can be used for reference to determine order: https://github.com/dotnet/source-build-reference-packages/pull/740.

**Comment by v-wuzhai** [comment]
@mthalman I have fixed above comments, could you help to merge this?

**Review by mthalman — APPROVED** [review]

**Pull request merged**